<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Splitter</class>
 <widget class="QWidget" name="Splitter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>520</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Gothic</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Splitter</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <widget class="QPushButton" name="button_ifo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose .IFO</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_vro">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose .VRO</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>440</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QScrollArea" name="output">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>441</width>
     <height>161</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="output_contents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>439</width>
      <height>159</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="output_text">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>441</width>
       <height>161</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_split">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Split .VRO</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_output">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="selected_ifo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selected: NONE</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="selected_vro">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selected: NONE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>10</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinbox_offset">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.000100000000000</double>
   </property>
   <property name="value">
    <double>-0.299900000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_offset">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Split Offset</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>2</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_grouping">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Group clips by title</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hint_offset">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How many seconds to offset splitting by. &lt;/p&gt;&lt;p&gt;Useful as ffmpeg trimming isn't perfect so there may be a few frames of the next clip at the end of each clip.&lt;/p&gt;&lt;p&gt;(e.g. offset of -0.2937: two clips 12s and 15s --&amp;gt; 11.7063s and 14.7063s)&lt;/p&gt;&lt;p&gt;The default value of -0.2999 seconds is what I found to work well in my testing&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="text">
    <string>(?)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="hint_grouping">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Whether or not clips should be grouped into directories based on their titles or not (when possible)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="text">
    <string>(?)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_splitting">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Splitting</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_skip">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Skip existing clips</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hint_skip">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Whether or not existing clips in the chosen directory should be skipped&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="text">
    <string>(?)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="copyright">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>© 2025 spinningbanana</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTop|Qt::AlignmentFlag::AlignTrailing</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_ifo</sender>
   <signal>clicked()</signal>
   <receiver>Splitter</receiver>
   <slot>choose_ifo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_vro</sender>
   <signal>clicked()</signal>
   <receiver>Splitter</receiver>
   <slot>choose_vro()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_split</sender>
   <signal>clicked()</signal>
   <receiver>Splitter</receiver>
   <slot>split_vro()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>choose_ifo()</slot>
  <slot>choose_vro()</slot>
  <slot>split_vro()</slot>
 </slots>
</ui>
